src/js/components/search/filters/timePeriod/DateRange.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * DateRange.jsx
+ * Created by Emily Gullo 11/03/2016
+ **/
+
+import React from 'react';
+import DatePicker from '../../../sharedComponents/DatePicker';
```

Comment:
With the latest `dev` update, we can now provide import paths that use `src/js` as the base directory. So this would be `import Datepicker from 'components/sharedComponents/DatePicker';`

You can still reference relative to the current file when that is more efficient. Start *relative* imports with `./`, `../`, etc. Don't use leading slashes or dots when importing relative to the base source directory.